css/main.css

```diff
@@ -223,3 +228,7 @@
   text-transform: capitalize;
   margin-bottom: 15px;
 }
+
+.filters .dropdown-body .active {
+  visibility: visible;
+}
```

Comment:
Please combine your selectors to one because they are the same: 

```css
.filters .dropdown-body .active {
  background-color: rgb(219, 234, 176);
}
```

And : 

```css
.filters .dropdown-body .active {
  visibility: visible;
}

```